apps/betterangels-e2e/.detoxrc.json

```diff
@@ -53,7 +64,7 @@
     "emulator": {
       "type": "android.emulator",
       "device": {
-        "avdName": "Pixel_4a_API_30"
+        "avdName": "TestDevice"
```

Comment:
The `avdName` seems brittle. 
In theory we should be able to instruct detox to create an android device based on configs, but I couldn't make it work initially. So would need to revisit this if we decide to continue with detox.